### PR TITLE
Add ignore option to commit_version_bump action.

### LINKED ISSUE #5510: Add ignore option to commit_version_bump action
It would be nice to have a ignore option in the `commit_version_bump` action to be able to ignore plist file from being committed.

I've already implemented this [here](https://github.com/syrmo/fastlane/blob/master/fastlane/lib/fastlane/actions/commit_version_bump.rb#L58) for our own project.

The reason is that we used different Info.plist files that are autogenerated depending on the build scheme used. This Info.plist files are generated from the normal Info.plist but we apply some modifications depending on the scheme. We ignore this autogenerated files.

### PR DESCRIPTION
**Pull request by guidomb**
Fixes #5510 

The ignore option is an optional regular expression that, if
provided, is used to filter the list of plist files to be
committed.

### CODE REVIEW

File: fastlane/lib/fastlane/actions/commit_version_bump.rb [hunk 1]
```diff
@@ -55,6 +55,14 @@ def self.run(params)
         end
         # rubocop:enable Style/MultilineBlockChain
 
+        # Removes .plist files that matched the given expression in the 'ignore' parameter
+        ignore_expression = params[:ignore]
+        if ignore_expression
+          info_plist_files.select! do |info_plist_file|
+            ! info_plist_file.match(ignore_expression)
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/commit_version_bump.rb]: Please remove the space here

**Comment by KrauseFx** [comment]
This looks great, thanks. Unfortunately I can't merge as long as the styling issue is here, could you remove the space, and let me know when you had the time to update the PR? 👍

**Comment by guidomb** [comment]
@KrauseFx Done

**Comment by KrauseFx** [comment]
Perfect, thanks @guidomb 👍

**Pull request merged**